src/__tests__/http-test.ts

```diff
@@ -123,7 +123,8 @@ describe('GraphQL-HTTP tests for express', () => {
 });
 
 describe('GraphQL-HTTP tests for restify', () => {
-  runTests(() => {
+  runTests(async () => {
+    const restify = await import('restify');
```

Comment:
@danielrearden Can you move it inside `describe` and make only `describe` async?